homebrew_challenge/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import "./Footer.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import {
+  PAGE_PUBLIC_DASHBOARD,
+  PAGE_DATA_SOURCES,
+  PAGE_ABOUT_US
+} from "../../PageConsts";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faLinkedinIn } from "@fortawesome/free-brands-svg-icons";
+import { faTwitter } from "@fortawesome/free-brands-svg-icons";
+
+const Footer = ({ setCurrentPage }) => {
+  function sitemapEntry(key, text) {
+    return (
+      <div className="entry link" onClick={() => setCurrentPage(key)}>
+        {" "}
+        <div
+          className="entry link"
+          id="entry-link"
+          onClick={() => setCurrentPage(key)}
+        >
+          {text}
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <footer>
+      <Container fluid className="font-small">
+        <Row className="p-1">
+          <Col
+            sm={12}
+            md={12}
+            className="p-2 d-flex justify-content-left sta-logo"
+          >
+            <a
+              href="https://www.scottishtecharmy.org/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="STABanner.png"
+                alt="Scottish Tech Army"
+                width="270"
+                height="50"
+              />
+            </a>
+          </Col>
+          <Col
+            sm={12}
+            md={12}
+            className="sitemap-container d-flex justify-content-between"
+          >
+            <div className="sitemap">
+              <div className="title">DASHBOARDS</div>
+              {sitemapEntry(PAGE_PUBLIC_DASHBOARD, "Summary Dashboard")}
+              <div className="entry link">
+                <a
+                  className="entry link"
+                  href="#"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Regional Dashboard
```

Comment:
This is for me to add? No problem

Reply:
yes - I just put in a placeholder for meantime